examples/web-wagmi/src/inbox/components/MessagesCard.tsx

```diff
@@ -0,0 +1,39 @@
+import { Conversation, DecodedMessage, useMessages, useStreamMessages } from '@xmtp/react-sdk';
+import { useCallback, useEffect, useState } from 'react';
+
+type MessagesCardProps = {
+  conversation: Conversation;
+};
+
+export function MessagesCard({ conversation }: MessagesCardProps) {
+  const { messages } = useMessages(conversation);
```

Comment:
Curious why xmtp doesn't expose a hook that always returns the messages by subscribing to the steam to reduce the API surface for the consumers.

Reply:
I guess you don't always want to subscribe, sometimes just fetch messages and that's all